### PR TITLE
bump rebar3 to 3.14.3

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

**Comment by getong** [comment]
https://travis-ci.org/github/erlang/docker-erlang-otp/jobs/747216096
is not passed, and it is not related to the changed code. But it is better to format the code as usual.

**Comment by tsloughter** [comment]
Well damn... Do you understand what is wrong? I see the error is `The command "docker build --pull -t "$image" "${VARIANT:-.}"" exited with 127.`

**Review by getong — CHANGES_REQUESTED** [review]

**Comment by getong** [comment]
> Well damn... Do you understand what is wrong? I see the error is `The command "docker build --pull -t "$image" "${VARIANT:-.}"" exited with 127.`

It might be something network wrong, you can make a new commit, and make the ci build again. I do not have the priviledge to make the ci build again.

File: 23/Dockerfile [hunk 1]
```diff
@@ -56,8 +56,8 @@ RUN set -xe \
 	&& rm -rf /usr/src/rebar-src
 
 RUN set -xe \
-	&& REBAR3_DOWNLOAD_URL="https://github.com/erlang/rebar3/archive/${REBAR3_VERSION}.tar.gz" \
-	&& REBAR3_DOWNLOAD_SHA256="19fcdc73debb90d405864f728e188cbc5b61c3939b911e58c0b59bf1619c4810" \
+    && REBAR3_DOWNLOAD_URL="https://github.com/erlang/rebar3/archive/${REBAR3_VERSION}.tar.gz" \
```
[the remark this answers is not in the document]
Comment (tsloughter) [reply 23/Dockerfile:59]: oh dammit, my emacs must be doing that. I thought I caught it and replaced with a tab.

[the remark this answers is not in the document]
Comment (tsloughter) [reply 23/Dockerfile]: Fixed.

**Comment by tsloughter** [comment]
Looks like travis is stuck queued... hopefully it goes through soon.

**Comment by tsloughter** [comment]
@getong looks like it'll finish this time.

You then send a PR to a docker repo, right? Is there a script that generates that patch or do you manually make the changes and send?

**Comment by getong** [comment]
I do it manually:
```
./generate-stackbrew-library.sh > ../official-images/library/erlang
```
and then open a pr.

**Pull request merged**